### ISSUE TITLE
Contributor roles - investigate default mapping to "Other role"

### ISSUE THREAD

**Issue by gneidhardt** [galterlibrary/digital-repository#1102]
Likely a post-migration project, but a reviewer brought up that it seemed like contributors by default got "Other role" which might not be accurate. Also brought up that field is not labeled as "contributor" so if you only see "other role" it's a bit confusing. Example:
https://digitalhub.northwestern.edu/files/c8b7baf2-8c94-4422-931b-bb290db7d549	https://vtfsmghslrepo02.fsm.northwestern.edu/records/dwj70-xg039

**Comment by fenekku** [comment]
Agreed, it's not very clear here. On the CERN demo site it can look ok-ish (e.g., https://inveniordm-qa.web.cern.ch/records/pgwhf-vrz28) but it probably wouldn't hurt to have "Contributors" as the section header.

Placing this in digitalhub-2
